membership-token/js/test/create-store.test.ts

```diff
@@ -36,47 +38,3 @@ test('create-store: success', async (t) => {
 
   assertConfirmedTransaction(t, createStoreRes.txConfirmed);
 });
-
-test('create-store: bad name', async (t) => {
```

Comment:
why is it removed?

Reply:
https://github.com/metaplex-foundation/metaplex-program-library/pull/151#discussion_r792420237